LICENSES/LLVM-exception.txt

```diff
@@ -0,0 +1,5 @@
+LLVM Exceptions to the Apache 2.0 License 
```

Comment:
Why do we add the file LLVM-exception.txt?

This was not in the original crix repository, and I don't see it being referenced anywhere.

Reply:
This is an artifact of an idea that I had for a brief period of time. Will be removed.